src/main/java/uk/gov/hmcts/cmc/claimstore/documents/content/ClaimantResponseContentProvider.java

```diff
@@ -77,12 +77,16 @@ public ClaimantResponseContentProvider(
                 content.put("defendantAdmissionAccepted", "I accept this amount");
                 ResponseAcceptation responseAcceptation = (ResponseAcceptation) claimantResponse;
                 content.putAll(responseAcceptationContentProvider.createContent(responseAcceptation));
-                content.put("formaliseOption", responseAcceptation.getFormaliseOption().getDescription());
 
-                claim.getTotalAmountTillDateOfIssue().ifPresent(totalAmount -> content.put("totalAmount",
-                    formatMoney(totalAmount.subtract(claimantResponse.getAmountPaid().orElse(BigDecimal.ZERO)))));
+                claim.getTotalAmountTillDateOfIssue().ifPresent(
+                    totalAmount -> content.put("totalAmount", formatMoney(totalAmount.subtract(claimantResponse.getAmountPaid().orElse(BigDecimal.ZERO))))
+                );
 
-                addFormalisedOption(claim, content, responseAcceptation);
+                if (responseAcceptation.getFormaliseOption().isPresent()) {
```

Comment:
Could also be clever with the lambdas here, but (without trying it out) I think it'll be cleaner as it is.